src/userConfig_sample.h

```diff
@@ -93,5 +98,8 @@ enum MACHINE {
 // PID Parameters (not yet in Web interface)
 #define EMA_FACTOR 0.6             // Smoothing of input that is used for Tv (derivative component of PID). Smaller means less smoothing but also less delay, 0 means no filtering
 
-#define TEMPSENSOR 2               // Temp sensor type: 1 = DS18B20, 2 = TSIC306
+#define TEMP_SENSOR 2               // Temp sensor type: 1 = DS18B20, 2 = TSIC306
+
+// Log level for serial console
+#define LOGLEVEL Logger::Level::INFO
```

Comment:
Should still include a comment for other options here (this file still kind of an end user config file, so most are likely not aware of log levels, also not which are relevant here)